.github/workflows/publishing.yml

```
@@ -768,6 +769,7 @@ jobs:
       - name: Install and run sanity checks
         shell: bash
         run: |
+          if [ -f ~/.bashrc ]; then . ~/.bashrc; fi
```

I'm wondering if that would source `.bashrc` twice in some platforms, which might be confusing to debug in the future. The bash man page offers an alternative that might work across a few different platforms better:
```
       When bash is started non-interactively, to run a shell script, for example, it  looks  for  the
       variable  BASH_ENV  in the environment, expands its value if it appears there, and uses the ex‐
       panded value as the name of a file to read and execute.  Bash behaves as if the following  com‐
       mand were executed:
              if [ -n "$BASH_ENV" ]; then . "$BASH_ENV"; fi
       but the value of the PATH variable is not used to search for the filename.
```

```suggestion
        env: |
          BASH_ENV: ~/.bashrc
        run: |
```
I tested this locally on my machine, and it works, but I can't guarantee that I got the yml syntax correct.